packages/microsoft-reactnative-sampleapps/windows/SampleLibraryCPP/DebugHelpers.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "DebugHelpers.h"
+
+namespace SampleLibraryCPP {
+
+void DebugWriteLine(const std::string &name, const std::string &methodName, const std::string &arg) {
+  std::string output = name;
+  output.append("::");
+  output.append(methodName);
+  output.append("(");
+  output.append(arg);
+  output.append(")\n");
+  OutputDebugStringA(output.c_str());
```

Comment:
I prefer output=name + "::" + methodName + ...
It will be more readable, also I didn't see benefit of append over this.

Reply:
Fixed.